pool/pool.go

```diff
@@ -1301,12 +1301,12 @@ func NewPool(options InitParameters) (*Pool, error) {
 		return nil, err
 	}
 
-	cache, err := newCache()
+	cache, err := newCache(defaultSessionCacheSize)
 	if err != nil {
 		return nil, fmt.Errorf("couldn't create cache: %w", err)
 	}
 
-	remoteKeysCache, err := newCache()
+	remoteKeysCache, err := newCache(len(nodesParams))
```

Comment:
Maybe this can be stored in `clientWrapper` or in array of some kind? It's clearly 1:1 related to nodes, so LRU is not really appropriate.

Reply:
I took LRU because it already has a mechanism to check token expirations, and remove tokens on node errors. I think `clientWrapper` may be significantly reworded or even removed in the next iteration because used for old methods.
For the new methods `clientWrapper` because we works with plain SDK client

Reply:
We need to take this 1:1 relation in some way, currently you have to generalize the cache to make it work in both cases.